doc/flame/components.md

```diff
@@ -498,47 +498,61 @@ Future<void> onLoad() async {
 }
 ```
 
-If you are not using `FlameGame`, don't forget this component needs to be updated, because the
-animation object needs to be ticked to move the frames.
+If you are not using `FlameGame`, don't forget this component needs to be updated, because a ticker
```

Comment:
Not new for this PR, but I don't think we need to mention cases not using it as a component for every separate component and then we can just mention it once that in the overview for components.

Reply:
Agreed. Probably it can even be a separate section altogether. Something like "Things to consider when not using `FlameGame`"

Reply:
Oh, just to be clear, is this change expected in this PR?😅

Reply:
Whatever you feel like, it would be faster in this PR, but it has nothing to do with this PR so we'll definitely merge it anyways. 😅